tests/test_crypto/test_fetchai.py

```diff
@@ -103,7 +101,14 @@ def test_generate_nonce():
     ), "The len(nonce) must not be 0 and must be hex"
 
 
-@pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS)
+def test_get_address_from_public_key():
+    """Test the address from public key."""
+    fet_crypto = FetchAICrypto()
+    address = FetchAIApi.get_address_from_public_key(fet_crypto.public_key)
+    assert address == fet_crypto.address, "The address must be the same."
+
+
+# @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS)
```

Comment:
Currently, some tests are failing due to missing support for wealth creation on new testnet.

Reply:
ok